README.md

```diff
@@ -17,7 +17,7 @@
   <br /><br />
   Bearer CLI is a static application security testing (SAST) tool that scans your source code and analyzes your data flows to discover, filter and prioritize security and privacy risks.
   <br /><br />
-  Currently supporting: <strong>JavaScript/TypeScript</strong> (GA), <strong>Ruby</strong> (GA), <strong>PHP</strong> (GA), <strong>Java</strong> (GA), <strong>Go</strong> (GA), <strong>Python</strong> (Beta) - <a href="https://docs.bearer.com/reference/supported-languages/">Learn more</a>
+  Currently supporting: <strong>JavaScript/TypeScript</strong> (GA), <strong>Ruby</strong> (GA), <strong>PHP</strong> (GA), <strong>Java</strong> (GA), <strong>Go</strong> (GA), <strong>Python</strong> (GA) - <a href="https://docs.bearer.com/reference/supported-languages/">Learn more</a>
```

Comment:
@gotbadger Since they're all GA now, maybe we want to remove the "(GA)" here for simplicity? It will still be in the docs.

Reply:
yeah lets do that

Reply:
^ https://github.com/Bearer/bearer/pull/1641/commits/a08b1582489f628e92103197e18248c0be70fdb3